import_and_export_data/guides/create-connector.rst

```diff
@@ -160,8 +161,8 @@ There are actually two forms for each job: one for edit mode and one for view mo
 
 For our form we'll need to copy:
 
-- ``vendor/akeneo/pim-community-dev/src/Akeneo/Platform/Bundle/UIBundle/Resources/config/form_extensions/job_instance/csv_product_export_edit.yml`` to ``src/Acme/Bundle/NotifyConnectorBundle/Resources/config/form_extensions/csv_product_export_notify_edit.yml``
```

Comment:
Not sure if we should change this line. It depends if the installation is done with the Standard or the Dev edition.